### PR TITLE
docs: dev setup docs, add pytest as a dev dependency

### PR DESCRIPTION
**Pull request by iloveitaly**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -101,6 +101,19 @@ Dependencies
...
+Development
+===========
+
+Here's how to setup your development environment:
+
+.. code-block:: shell
+
+    python -m venv .venv
+    . .venv/bin/activate
+    pip install -e ".[dev]""
```
Comment (edelooff) [inline README.rst:113]:
```suggestion
    pip install -e ".[dev]"
```

File: setup.py [hunk 2]
```diff
@@ -22,6 +22,9 @@ def contents(filename):
     packages=["sqlalchemy_json"],
     python_requires=">= 3.6",
     install_requires=["sqlalchemy>=0.7"],
+    extras_require={
+        'dev': ['pytest']
+    },
```
Comment (edelooff) [inline setup.py:27]:
```suggestion
    extras_require={
        "dev": ["pytest"]
    },
```

**Review by edelooff — COMMENTED** [review]
Looks good, if you could fix the extra quote and fix the quoting in the setup I'm happy to merge it in.

**Comment by iloveitaly** [comment]
@edelooff just updated!

**Comment by edelooff** [comment]
Thanks, squashed the fix and merged.

**Pull request merged**